docs/0.1/grid_product_catalog_family_specification.md

```diff
@@ -0,0 +1,496 @@
+products in this Grid network.
+
+The Grid Product Catalog depends on the existence of this schema. Therefore, all
+Grid Product Catalog transactions will be invalid under the following
+circumstances:
+* There is no schema in state with the name "Catalog Product"
+* The schema with the name "Catalog Product" does not have the `catalog_id` or
+  `status` property definitions.
+
+An example "Product Catalog" schema definition is shown below:
+
+```yaml
+- name: "Catalog Product"
+  description: "Schema defining a catalog product"
+  owner: "ACME Corporation"
+  properties:
+    - name: "catalog_id"
+      data_type: STRING
+      description: "The ID of the catalog that this catalog product belongs to"
+      required: true
+    - name: "status"
+      data_type: ENUM
+      description: "The current status of the catalog product"
+      enum_options: ["ACTIVE", "INACTIVE", "DISCONTINUED"]
+      required: true
+    - name: "price"
+      data_type: STRING
+      description: "The price of the product"
+      required: true
+    - name: "return_policy"
+      data_type: STRING
+      description: "A description of the return policy for this product"
+      required: false
+```
+
+Note that this schema is a valid product catalog schema because it has the name
+"Catalog Product", and it has the "catalog_id" and "status" required properties
+with the correct data types. This is all that is required for a minimal catalog
+product schema definition.
+
+This example also defines several additional requirements for catalog products:
+"price" and "return policy". In this example Grid network, all catalog products
+must include the price, since this property is required. They can optionally
+include a description of the return policy, since this is listed as an optional
+property in the schema.
+
+## Addressing
+
+In order to uniquely locate Grid Product Catalog state objects in the
+Merkle-Radix state system, an address must be constructed which identifies the
+storage location of the object.
+
+All state address for Grid Product Catalog objects begin with the Grid
+namespace: `621dee`. They are further prefixed with the reserved namespace
+prefix for Grid Product Catalog: `03`.
+
+After the namespace prefix, the next two characters of a Grid Product Catalog
+object's address are a string based on the object's type:
+
+* Catalog: `00`
+* Catalog Product: `01`
+
+The next 60 characters are determined by the object's type:
+
+* Catalog: The first 44 characters of a SHA-512 hash of the catalog ID, followed
+  with 16 hex-character zeroes to accommodate potential future storage
+  associated with the GS1 Catalog representation.
+* Catalog Product: The first 44 characters of a SHA-512 hash of the catalog ID
+  that the product is associated with, followed by the 14 character GTIN of the
+  product, followed by 2 hex-character zeroes to accommodate potential future
+  storage associated with the catalog Product representation.
```

Comment:
Other than Shawn's small quibble, this section is great.  Thanks for the clear information!